client/lib/upgrades/constants.js

```diff
@@ -16,40 +25,26 @@ module.exports.action = keyMirror( {
 	EMAIL_FORWARDING_FETCH: null,
 	EMAIL_FORWARDING_FETCH_COMPLETED: null,
 	EMAIL_FORWARDING_FETCH_FAILED: null,
-	DOMAIN_ENABLE_LOCKING_COMPLETED: null,
-	DOMAIN_ENABLE_PRIVACY_PROTECTION_COMPLETED: null,
-	DOMAIN_TRANSFER_ACCEPT_COMPLETED: null,
-	DOMAIN_TRANSFER_CODE_REQUEST_COMPLETED: null,
-	DOMAIN_TRANSFER_DECLINE_COMPLETED: null,
-	FETCH_DNS: null,
-	FETCH_GOOGLE_APPS_USERS: null,
+	GOOGLE_APPS_USERS_FETCH: null,
+	GOOGLE_APPS_USERS_FETCH_COMPLETED: null,
+	ICANN_VERIFICATION_RESEND_COMPLETED: null,
 	NAMESERVERS_FETCH: null,
 	NAMESERVERS_FETCH_COMPLETED: null,
 	NAMESERVERS_FETCH_FAILED: null,
 	NAMESERVERS_UPDATE_COMPLETED: null,
-	OPEN_CART_POPUP: null,
 	PRIMARY_DOMAIN_SET: null,
 	PRIMARY_DOMAIN_SET_COMPLETED: null,
 	PRIMARY_DOMAIN_SET_FAILED: null,
-	PURCHASES_PRIVATE_REGISTRATION_CANCEL: null,
-	PURCHASES_PRIVATE_REGISTRATION_CANCEL_COMPLETED: null,
-	PURCHASES_PRIVATE_REGISTRATION_CANCEL_FAILED: null,
+	PRIVACY_PROTECTION_CANCEL: null,
+	PRIVACY_PROTECTION_CANCEL_COMPLETED: null,
+	PRIVACY_PROTECTION_CANCEL_FAILED: null,
+	PRIVACY_PROTECTION_ENABLE_COMPLETED: null,
```

Comment:
@stephanethomas can you open a new issue for that? I'm guessing this falls under Cobalt's realm, so pinging @aidvu.